src/content/reference/web-interface/tenant-cluster-labelling.md

```diff
@@ -5,9 +5,6 @@ date: 2020-11-18
 last_review_date: 2020-05-29
 layout: subsection
 weight: 20
-aliases: [
-    "/reference/web-interface/tenant-cluster-labelling/",
-]
```

Comment:
Removed, as it does not make sense. Caused a warning like "two pages want to write to the same path".